src/main/webapp/content/scss/global.scss

```diff
@@ -223,3 +223,34 @@ ui bootstrap tweaks
  }
 
 /* jhipster-needle-scss-add-main JHipster will add new css style */
+
+/* common css classes start */
+.addIcon {
+    color: green;
+    margin-right:8px
+}
+.deleteIcon {
+    color: red;
+    margin-right:8px
+}
+.editIcon {
+    margin-right:8px
+}
+.exchangeIcon {
```

Comment:
We should try to merge the same styles together and leave comments

Reply:
I'll check common styles in the project and move them to here.